### PR TITLE
Prod user pool config

### PR DESCRIPTION
**Pull request by ap-justin**
Ticket(s):
-

## Explanation of the solution
* switch to literal config object instead of auto-generated `aws-exports` 
* test prod auth flow

 @SovereignAndrey 
need to add redirectURL `gc8hpcg5cpes-dev.auth.us-east-1.amazoncognito.com` in google developer console for oauth to work in prod

## Instructions on making this work
* set env `ENVIRONMENT=PRODUCTION` and try to register - should still work 

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/store/amplify-config.ts
Comment (ap-justin) [inline src/store/amplify-config.ts:6]: this is only temporary as current `prod` doesn't have any auth in it. We will eventually switch back to auto-generated `aws-exports` file where `master` would have it's own config from `amplify pull prod` and `staging` from `amplify pull dev`

**Review by misicnenad — COMMENTED** [review]
Tentative approval from me, but waiting on @SovereignAndrey to review

**Review by SovereignAndrey — APPROVED** [review]
@ap-justin Looks good! Having issues logging into the AP GCP Account. Will work to have google unblock it and set prod the redirect URL in a separate prod OAuth project.

**Pull request merged**
